net/BigBuffers.Tests/BigBuffers.Tests.csproj

```
@@ -6,6 +6,13 @@
     <IsPackable>false</IsPackable>
 
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <Authors>Stirling Labs Team &amp; Contributors</Authors>
+    <CollectCoverage>true</CollectCoverage>
+    <CoverletOutputFormat>cobertura</CoverletOutputFormat>
+    <CoverletOutput>../../coverage/</CoverletOutput>
```

```suggestion
    <CoverletOutput>../coverage/</CoverletOutput>
```

This should probably just be for the solution directory; there will be other coverage information from other projects.